blueprint/api.apib

```diff
@@ -2174,6 +2245,33 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(Slugged Route Organization Resource)
 + include JSON API Version
 
+## Stripe Plan Attributes (object)
++ amount: 999 (number) - Describes the amount of a plan in cents.
++ created: `2016-07-08T03:03:51.967Z` (string)
++ id-from-stripe: `monthly-donation-1`
```

Comment:
In addition to ticks surrounding this, also think we need to have this be `(string, required)`